test/util/test.js

```diff
@@ -25,6 +26,15 @@ type CreateTest = (typeof sinon) & {
     tearDown(() => void): void,
 };
 
+// Add function for floating point equality checking.
+tap.Test.prototype.almostEqual = function(found, wanted, epsilon, message) {
+    return this.ok(
+        almostEqual(found, wanted, epsilon),
+        message || "should be almost equal",
+        {found, wanted, compare: "~="}
+    );
+};
+
```

Comment:
See also https://github.com/mapbox/mapbox-gl-js/blob/master/test/util/index.js#L33 which has the same purpose and is used throughout various unit tests. Needs to be de-duplicated. I prefer the explicit version rather than a prototype augment to make #9241 easier in the future.

Reply:
Ah, I didn't find this method.